front/app/components/admin/PostManager/components/PostPreview/Idea/AdminIdeaContent.tsx

```diff
@@ -0,0 +1,331 @@
+`;
+
+const BodySectionTitle = styled.h2`
+  font-size: ${(props) => props.theme.fontSizes.m}px;
+  font-weight: 400;
+  line-height: 28px;
+`;
+
+const StyledMap = styled(DropdownMap)`
+  margin-bottom: 40px;
+`;
+
+const StyledOfficialFeedback = styled(OfficialFeedback)`
+  margin-top: 70px;
+`;
+
+const Right = styled.div`
+  flex: 2;
+  position: sticky;
+  top: 80px;
+  align-self: flex-start;
+  color: ${colors.primary};
+  font-size: ${fontSizes.base}px;
+  line-height: 19px;
+`;
+
+const BudgetBox = styled.div`
+  margin-top: 25px;
+  width: 100%;
+  height: 95px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 5px;
+  position: relative;
+  border-radius: 5px;
+  background: ${colors.background};
+  border: solid 1px ${colors.primary};
+  font-size: ${fontSizes.l}px;
+  font-weight: 500;
+`;
+
+const Picks = styled.div`
+  margin-top: 15px;
+  display: flex;
+  font-size: ${fontSizes.base}px;
+  align-items: center;
+`;
+
+interface Props {
+  ideaId: string;
+  closePreview: () => void;
+  handleClickEdit: () => void;
+}
+
+const AdminIdeaContent = ({ handleClickEdit, closePreview, ideaId }: Props) => {
+  const { data: idea } = useIdeaById(ideaId);
+  const { data: project } = useProjectById(
+    idea?.data.relationships.project.data.id
+  );
+  const { formatMessage } = useIntl();
+  const localize = useLocalize();
+  const { data: ideaImages } = useIdeaImages(ideaId);
+  const { data: ideaFiles } = useIdeaFiles(ideaId);
+  const { mutate: deleteIdea } = useDeleteIdea();
+  const { data: phases } = usePhases(project?.data.id);
+
+  if (!idea || !project) return null;
+
+  const handleClickDelete = (processType: ProcessType) => () => {
+    const deleteConfirmationMessage = {
+      continuous: messages.deleteInputConfirmation,
+      timeline: messages.deleteInputInTimelineConfirmation,
+    }[processType];
+
+    if (window.confirm(formatMessage(deleteConfirmationMessage))) {
+      deleteIdea(idea.data.id, { onSuccess: closePreview });
+    }
+  };
+
+  const currentPhase = getCurrentPhase(phases?.data);
+  const ideaTitle = localize(idea.data.attributes.title_multiloc);
+  const ideaImageLarge =
+    ideaImages && ideaImages.data.length > 0
+      ? ideaImages.data[0].attributes.versions.large
+      : null;
+  const ideaGeoPosition = idea.data.attributes.location_point_geojson || null;
+  const ideaAddress = getAddressOrFallbackDMS(
+    idea.data.attributes.location_description,
+    idea.data.attributes.location_point_geojson
+  );
+  // AuthorId can be null if user has been deleted
+  const authorId = idea.data.relationships.author.data?.id || null;
+  const proposedBudget = idea.data.attributes.proposed_budget;
+  const processType = project.data.attributes.process_type;
+  const allowAnonymousParticipation =
+    project.data.attributes.allow_anonymous_participation ||
+    currentPhase?.attributes.allow_anonymous_participation;
+
+  return (
```

Comment:
Minor: Looks like this component was just moved but Github is not showing it as such. We could break down some of the sections in this component to smaller ones but I'll leave it to you since these are technically not new additions